Dockerfile

```diff
@@ -1,11 +1,11 @@
-ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
+ARG APP_INSIGHTS_AGENT_VERSION=3.4.13
```

Comment:
this should be added so that renovate can automatically keep this up to date:
```suggestion
# renovate: datasource=github-releases depName=microsoft/ApplicationInsights-Java
ARG APP_INSIGHTS_AGENT_VERSION=3.4.13
```

Reply:
done